pkg/services/rpcsrv/server.go

```diff
@@ -2674,7 +2676,11 @@ func (s *Server) unsubscribeFromChannel(event neorpc.EventID) {
 	}
 }
 
+// handleSubEvents processes Server subscriptions until Shutdown. Upon
+// completion signals to subEventCh channel.
 func (s *Server) handleSubEvents() {
+	defer close(s.subEventCh)
```

Comment:
Move it to the end of the method and fix the comment there?

Reply:
moved